pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -1375,12 +1350,56 @@ class WorldSessionActor extends Actor with MDCContextAware {
         RequestSanctuaryZoneSpawn(player, zone_number)
       }
 
+    case Zone.Ground.ItemOnGround(item, pos, orient) =>
+      item.Position = pos
+      item.Orientation = Vector3(0,0, orient.z) //only one kind of rotation is important
```

Comment:
The first field is roll and the second is pitch.  No items roll or pitch when dropped.

Reply:
That makes sense, but I'd note that in the comment so it's easy to see why at a glance. Perhaps in the future it would actually make sense to have certain types of items (e.g. dropped weapons) lie on their side, instead of directly upwards which is a bit odd when you think about it.

Reply:
I'll just be happy when I can get all items laying flat on the ground and not sunk into it (like that MCG that almost completely vanishes when dropped).